src/hooks/rolebot/rolebot.ts

```diff
@@ -0,0 +1,119 @@
+import { Client, Role, MessageReaction, GuildMemberRoleManager, User, PartialUser } from 'discord.js';
+import { Hook } from '../../utils/hook';
+import { DiscordService } from '../../services/app/discord_service';
+
+/* examples of valid role messages:
+
+react with :CelestiaFire: to get the role @testRole2
+react with 👍 to get the role @testRole1
+
+*/
+const REGEX = /^react with ?(?:<:([^:]+):\d+>|(\S{1,4})) ?to get the role ?<@&(\d+)>$/i;

```

Comment:
I can't seem to get this to work on regexr, but I believe you tested them, so I'm mostly trying to figure out where I'm messing up,